mathics/builtin/list/associations.py

```diff
@@ -16,13 +16,16 @@
 
 from mathics.core.expression import Expression
 from mathics.core.atoms import Integer
-from mathics.core.symbols import Symbol, SymbolList
+from mathics.core.symbols import Symbol, SymbolList, SymbolTrue
 from mathics.core.systemsymbols import (
     SymbolAssociation,
     SymbolMakeBoxes,
 )
 
 
+SymbolRowBox = Symbol("RowBox")
```

Comment:
I think this should be in mathics.core.systemsymbos

Reply:
Done.